control/celeryapp.py

```diff
@@ -28,7 +28,7 @@
     rabbitmq_broker=auth_dict['rabbitmq_broker']
 )
 
-app = Celery(auth_dict['celery_app_name'], broker=CEL_BROKER, backend=CEL_BACKEND, include=[])
+app = Celery(auth_dict.get('celery_app_name', None), broker=CEL_BROKER, backend=CEL_BACKEND, include=[])
```

Comment:
@muhammad-ammar I was trying to move the celery information out of static-config since it's fungible, but I see your point about sandboxes below, so I'll put it back.